ranch/priority.go

```diff
@@ -91,21 +93,22 @@ func newRequestQueue() *requestQueue {
 
 // update updates expiration time is updated if already present,
 // add a new requestID at the end otherwise (FIFO)
-func (rq *requestQueue) update(requestID string, newExpiration time.Time) bool {
+func (rq *requestQueue) update(requestID string, newExpiration time.Time) (bool, time.Time) {
 
 	rq.lock.Lock()
 	defer rq.lock.Unlock()
 	req, exists := rq.requestMap[requestID]
+	createdAt := time.Now()
 	if !exists {
-		req = request{id: requestID}
+		req = request{id: requestID, createdAt: createdAt}
```

Comment:
Please change to just `request{id: requestID, createdAt: time.Now()}`, to:
* Make the code more readablye by not defining a variable that is at most accessed once
* Avoid defining a variable that is potentially never needed

Reply:
Updated.